torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4108,10 +4161,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypesIfCUDA=complex_types(),
                    dtypesIfROCM=complex_types(),
                    supports_out=False,
-                   supports_autograd=False,
+                   test_conjugated_samples=False,
```

Comment:
Did you mean to clear out `supports_autograd` here?

Reply:
yeah `imag` supports autograd. not sure why it was set to `False`